docs/linux/dm.md

```diff
@@ -319,82 +323,73 @@ bitmaps.
-is indexed by logical chunk number and each mapping entry indicates the data
-zone storing the chunk data and optionally the zone number of a random zone
-used to buffer random modification to the chunk data.
-
-3. A set of blocks used to store bitmaps indicating the validity of blocks in
-the data zones follows the mapping table blocks. A valid block is a block that
-was writen and not discarded. For a buffered data zone, a block can be valid
-only in the data zone or in the buffer zone.
-
-To protect internal metadata against corruption in case of sudden power loss or
-system crash, two sets of metadata zones are used. One set, the primary set, is
-used as the main metadata set, while the secondary set is used as a log.
-Modified metadata are first written to the secondary set and the log so created
-validated by writing an updated super block in the secondary set. Once this log
-operation completes, updates in place of metadata blocks can be done in the
+1. **Super Block**: The first block of the first randomly-writable zone that is
+found contains the super block, which describes the amount and position (on
+the disk) of metadata blocks. 
+
+2. **Mapping-table blocks**: After the super block, there is a set of blocks 
+that describes the mapping of (1) the logical chunks of the target logical 
+device to (2) data zones. The mapping is indexed by logical chunk number, and 
+each mapping entry indicates the data zone that stores the chunk data. 
+It can also indicate the zone number of a random zone that is used to 
+buffer random modifications to the chunk data.
+
+3. **Bitmap-storage blocks**: After the mapping-table blocks, there is a set of 
+blocks used to store bitmaps that indicate the validity of blocks in 
+the data zones. A valid block is any block that was written and not discarded. 
+In a buffered data zone, a block can be valid only (1) in the data zone or (2)
+in the buffer zone.
+
+To protect internal metadata against corruption in cases of sudden power loss 
+or system crashes, two sets of metadata zones are used. One set (the primary 
+set) is used as the main metadata set. The other set (the secondary set) is 
+used as a log. Modified metadata are first written to the secondary set and 
+the log that is created by that act of writing to the secondary set is 
+validated by writing an updated super block in the secondary set. After this 
+log operation completes, updates in place of metadata blocks can be done in the
 primary metadata set, ensuring that one of the set is always correct.
-Flush operations are used as a commit point: upon reception of a flush
-operation, metadata activity is temporarily stopped, all dirty metadata logged
-and updated and normal operation resumed. This only temporarily delays write and
-discard requests. Read requests can be processed while metadata logging is
-executed.
+Flush operations are used as a commit point: when a flush operation is
```

Comment:
May be add a blank line before this sentence to make this a new paragraph.

Reply:
I've added a blank line before "Flush operations are used as a commit point: when a flush operation is"